ddtrace/ext/git.py

```diff
@@ -53,50 +53,51 @@
 log = get_logger(__name__)
 
 
+def _git_subprocess_cmd(cmd, cwd=None):
+    # type: (str, Optional[str]) -> str
+    """Helper for invoking the git CLI binary."""
+    git_cmd = subprocess.Popen(cmd.split(" "), stdout=subprocess.PIPE, stderr=subprocess.PIPE, cwd=cwd)
```

Comment:
Should the command by default use `git` rather than needing to have it be specified each time?

Reply:
Do you mean something like this?
```python
def _git_subprocess_cmd(cmd="git", cwd=None):
```

Reply:
> Or do you mean adding git as the base command so that any calls to this helper only need the actual commands rather than having to specify git ... each time?


this, yeah